10.5/cccp.yml

```diff
@@ -0,0 +1 @@
+job-id: mariadb-103-centos7
```

Comment:
I guess, it should be `mariadb-105-centos7`, right?

Reply:
Yes, fixing.